### DISCUSSION TITLE
I need the goreleaser install command.

### DISCUSSION THREAD

**Discussion by mondy** [goreleaser/goreleaser#2843] [Ideas, Issue Triage, and General discussions]
I want to `go install` using the builds flag described in `.goreleaser.yaml`.
I would like to have a `goreleaser install` command as well as `goreleaser build`.
It is troublesome to move the executable after `goreleaser build`.

**Comment by caarlos0** [comment]
`goreleaser build --single-target` will now copy the binary to the root folder

`goreleaser install` can be reproduced with `goreleaser build --single-target -o /usr/local/bin/foo`

all this in the next goreleaser version (v.1.5.0)

**Reply by mondy** [reply]
If possible, I want the output destination to be $GOPATH/bin automatically like `go install`.

**Reply by caarlos0** [reply]
You could use a makefile or wrap it within a shell script...
